src/App.tsx

```diff
@@ -344,9 +344,7 @@ function App() {
               oldAssetsEnoughToMigrate && <CallToAction setMigrationModalOpen={setMigrationModalOpen} />}
 
             <Switch>
-              {/* (keith): leave this route here temporarily to 
-              be able to reference the old dashboard */}
-              <Route exact path="/dashboard">
+              <Route path="/dashboard">
                 <TreasuryDashboard />
```

Comment:
We should add routes to these other pages so they're discoverable. 
Right now when a user clicks on any of these tabs, the route does not update.